.gitignore

```diff
@@ -8,3 +8,4 @@ test/keystore/
 test-keys/
 test/browser/bundle.js
 test/browser/bundle.js.map
+*.swp
```

Comment:
I think this should be part of your global gitignore as this is specific to your OS.

Reply:
Fixed